libraries/chain/include/eosio/chain/webassembly/interface.hpp

```diff
@@ -508,6 +508,30 @@ namespace webassembly {
          */
          bool is_account(account_name account) const;
 
+         /**
+          * Retrieves the code hash for an account, if any.
+          *
+          * The result is the packed version of this struct:
+          *
+          * struct {
+          *    uint64_t code_sequence;
+          *    fc::sha256 code_hash;
+          *    uint8_t vm_type;
+          *    uint8_t vm_version;
+          * } result;
+          *
+          * @ingroup authorization
+          * @param account - name of the account to check.
+          * @param struct_version - must be 0.
```

Comment:
For other similar functions, we accept any value for the version, and and include the version in the result.  The rationale for this is that it allows the contract and the system to negotiate the highest version that they both understand.